src/Implementations/Renderer.php

```diff
@@ -12,15 +13,33 @@
 
 class Renderer extends Environment implements RI
 {
+    /**
+     * @var string
+     */
+    private $testClass;
+
     /**
      * @param SplFileInfo $templates
+     * @param CI $composer
      */
-    public function __construct(SplFileInfo $templates)
+    public function __construct(SplFileInfo $templates, CI $composer)
     {
         parent::__construct(new FilesystemLoader($templates->getPathname()));
         $this->addTokenParser(new IncludeParser());
         $this->filterToUpperCamelCase();
         $this->setLexer(new Lexer($this));
+        $this->testClass = $composer->getTestClass();
+    }
+
+    /**
+     * @param string $name
+     * @param array $context
+     * @return string
+     */
+    public function render($name, array $context = array())
+    {
+        $context['config']['testcase'] = $this->testClass;
+        return parent::render($name, $context);
```

Comment:
Just for making sure, Twigs::render function returns a String?

Reply:
yes